ebean-test/src/test/java/misc/migration/v1_1/EBasic.java

```diff
@@ -41,6 +41,10 @@
   Integer id;
 
   @NotNull
+  @DbMigration(platforms = Platform.YUGABYTE, preAlter = {
+      "update ${table} set ${column} = ${default} where ${column} is null",
+      "commit transaction /* 1 */" })
```

Comment:
yugabyte needs commits here
@rbygrave what do you think, if the ddl generator will handle the case. e.g. here: https://github.com/ebean-orm/ebean/blob/master/ebean-ddl-generator/src/main/java/io/ebeaninternal/dbmigration/ddlgeneration/platform/BaseTableDdl.java#L130